spec/changelog.yml

```diff
@@ -2,7 +2,7 @@
 ## This file documents changes in the package specification. It is NOT a package specification file.
 ## Newer entries go at the bottom of each in-development version.
 ##
-- version: 3.0.0-next
+- version: 3.0.0-rc1
```

Comment:
Should we keep `3.0.0-next` and `2.13.1-next` version ? And update changelog in a separate PR ? 

I was thinking that maybe it could be done everything in this PR.